skbeam/core/roi.py

```diff
@@ -458,6 +459,67 @@ def circular_average(image, calibrated_center, threshold=0, nx=100,
     return bin_centers, ring_averages
 
 
+def construct_circ_avg_image(radii, intensities, centerdims=None,
+                             pixel_size=(1, 1)):
+    """ Constructs a 2D image from circular averaged data
+        where radii are given in units of pixels.
+        Normally, data will be taken from circular_average and used to
+        re-interpolate into an image.
+
+    Parameters
+    ----------
+    radii : 1D array of floats
+        the radii (must be in pixels)
+    intensities : 1D array of floats
+        the intensities for the radii
+    centerdims : 4 tuple of floats, optional
```

Comment:
yeah it's more sensible, thanks. both are still required, if one is missing the defaults are used for both